src/llamacrawl/services/readers/elasticsearch.py

```
@@ -269,37 +263,32 @@ def _load_with_pit(self, indices: list[str], query: dict[str, Any], limit: int |
             # Open Point-in-Time
             pit_response = self.es_client.open_point_in_time(
-                index=",".join(indices),
-                keep_alive="5m"
+                index=",".join(indices), keep_alive="5m"
             )
             pit_id = pit_response["id"]
 
             self.logger.debug(
-                f"Opened PIT: {pit_id}",
-                extra={"source": self.source_name, "pit_id": pit_id}
+                f"Opened PIT: {pit_id}", extra={"source": self.source_name, "pit_id": pit_id}
```

Ruff (0.14.0)

Logging statement uses f-string

(G004)